.github/workflows/build-backend.yml

```diff
@@ -12,23 +12,23 @@ on:
     types: [ opened, reopened ]
 
 jobs:
-  compliance:
-    runs-on: ubuntu-latest
-
-    steps:
-      - name: Checkout code
-        uses: actions/checkout@v3
-
-      - name: Advance Security Policy as Code
-        uses: advanced-security/policy-as-code@v2.4.1
-        with:
-          policy: it-at-m/policy-as-code
-          policy-path: default.yaml
-          token: ${{ secrets.GITHUB_TOKEN }}
-          argvs: "--disable-dependabot --disable-secret-scanning --disable-code-scanning --display"
+#  compliance:
```

Comment:
Warum ist das entfernt worden?

Reply:
weil der compliance job den man da von it-at-m erbt damals buggy war und alle workflows gefailed sind. ich würd das in main reinmergendamit sprint und main gleich sind und mich in der zukunft mal drum kümmern ob die jobs wieder funktionieren